Lib/profile.py

```diff
@@ -553,11 +553,13 @@ def main():
     import os
     from optparse import OptionParser
 
-    usage = "profile.py [-o output_file_path] [-s sort] scriptfile [arg] ..."
+    usage = "profile.py [-o output_file_path] [-s sort] [-m module |escriptfile] [arg] ..."
```

Comment:
Could you upgrade the documentation (Doc/library/profile.rst) and add a `..versionadded:: 3.8` ?

Reply:
Thanks! addressed 😄